site2/docs/security-extending.md

```diff
@@ -90,9 +90,9 @@ The following is the example for Broker authentication plugins:
 
 ## Authorization
 
-Authorization is the operation that checks whether a particular "role" or "principal" has a permission to perform a certain operation.
+Authorization is the operation that checks whether a particular "role" or "principal" has permission to perform a certain operation.
 
-By default, Pulsar provides an embedded authorization, though configuring a different one through a plugin is also an alternative choice.
+By default, Pulsar provides an embedded authorization provider, though configuring a different one through a plugin is also a choice.
```

Comment:
How about the following? You can take it as reference.

By default, you can use the embedded authorization provider provided by Pulsar. You can also configure a different authorization provider through a plugin.